src/AnnotationData.php

```diff
@@ -31,14 +31,14 @@ public function set($key, $value = '')
         return $this;
     }
 
-    public function append($key, $value = '')
+    public function append($key, $value = ''): void
     {
         $data = $this->offsetGet($key);
         if (is_array($data)) {
             $this->offsetSet($key, array_merge($data, $value));
         } elseif (is_scalar($data)) {
             $this->offsetSet($key, $data . $value);
         }
-        return $this;
+        // return $this;
```

Comment:
@greg-1-anderson this was a bandaid fix i wanted to highlight to you.. deprecation wants this method to be compatible with ArrayObject which is declared `void`, so it can't return anything.  Removing this return satisfies the deprecation but I think this means it will break anything downstream which is trying to `->chain()->calls()` with this.